file_manager/run_file.py

```diff
@@ -0,0 +1,96 @@
+import threading
+import os
+import traceback
+
+import pexpect
+
+from helpers.logger import create_logger
+
+
+FILE_PATH = '/home/username/Code_Docs/running_files/'
+DOCKER_IMAGES = {'python': 'code_docs_python'}
+
+
+run_file_logger = create_logger("run_file_logger")
+
+
+class RunFileThread(threading.Thread):
+    def __init__(self, file_name, file_content, programming_language, consumer=1):
+        super().__init__(name="run file")
+        self.filename = self.create_file(file_name, file_content)
+        self.docker_image = DOCKER_IMAGES[programming_language]
+        self.consumer = consumer
+        self.inputs = []
+        self.__close_force = False
+
+    def create_file(self, file_name, file_content):
+        generated_filename = FILE_PATH + file_name
+
+        if not os.path.exists(FILE_PATH):
+            os.makedirs(FILE_PATH)
+
+        with open(generated_filename, 'w') as file:
+            file.write(file_content)
+
+        return generated_filename
+
+    def delete_file(self):
+        try:
+            # run_file_logger.info(f"delete_file")
+            os.remove(self.filename)
+        except OSError:
+            # run_file_logger.info("file was deleted")
+            pass
+        
+    def add_input(self, file_input):
+        # run_file_logger.info(f"ADD INPUT {file_input}")
```

Comment:
is this necessary? and other comments too?

Reply:
is it better to delete?

Reply:
may be just decoment